### PR TITLE
Issue26858 tob tests examples

### PR DESCRIPTION
**Pull request by Malarg**
Handled failed to load example error

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

File: learning/tour-of-beam/frontend/integration_test/tour_page_test.dart [hunk 1]
```diff
@@ -1,3 +1,5 @@
+// ignore_for_file: avoid_print
```
Comment (alexeyinkin) [inline learning/tour-of-beam/frontend/integration_test/tour_page_test.dart:1]: After license.

File: playground/frontend/lib/main.dart [hunk 2]
```diff
@@ -31,6 +31,7 @@ import 'l10n/l10n.dart';
 import 'locator.dart';
 import 'playground_app.dart';
 
+//temp comment
```
Comment (alexeyinkin) [inline playground/frontend/lib/main.dart:34]: Delete.

File: playground/frontend/playground_components/lib/src/controllers/example_loaders/examples_loader.dart [hunk 3]
```diff
@@ -43,6 +47,8 @@ class ExamplesLoader {
   PlaygroundController? _playgroundController;
   ExamplesLoadingDescriptor? _descriptor;
 
+  static List failedToLoadExamples = List<String>.empty(growable: true);
```
Comment (alexeyinkin) [inline playground/frontend/playground_components/lib/src/controllers/example_loaders/examples_loader.dart:50]:
```suggestion
  static final failedToLoadExamples = <String>[];
```

File: playground/frontend/playground_components/lib/src/controllers/example_loaders/examples_loader.dart [hunk 4]
```diff
@@ -70,6 +76,7 @@ class ExamplesLoader {
   Future<void> load(ExamplesLoadingDescriptor descriptor) async {
     _descriptor = descriptor;
     final loaders = descriptor.descriptors.map(_createLoader).whereNotNull();
+    print('Loaders: ' + loaders.map((e) => e.descriptor.token).join(', '));
```
Comment (alexeyinkin) [inline playground/frontend/playground_components/lib/src/controllers/example_loaders/examples_loader.dart:79]: Delete.

File: playground/frontend/playground_components/lib/src/controllers/example_loaders/examples_loader.dart [hunk 5]
```diff
@@ -147,7 +154,12 @@ class ExamplesLoader {
     Example example;
     try {
       example = await loader.future;
-    } on Exception {
+    } on MultipleExceptions catch (ex) {
+      example = Example.empty(loader.sdk ?? Sdk.java);
+      _handleLoadException(loader, ex);
+    } on Exception catch (ex) {
+      example = Example.empty(loader.sdk ?? Sdk.java);
+      _handleLoadException(loader, ex);
```
Comment (alexeyinkin) [inline playground/frontend/playground_components/lib/src/controllers/example_loaders/examples_loader.dart:162]: Explain why we do not throw in one of the cases.

**Review by alexeyinkin — CHANGES_REQUESTED** [review]

File: playground/frontend/playground_components/lib/src/controllers/example_loaders/examples_loader.dart (hunk 5 above)
Comment (nausharipov) [reply playground/frontend/playground_components/lib/src/controllers/example_loaders/examples_loader.dart]: Is this relevant now?

**Comment by tvalentyn** [comment]
waiting on author

**Comment by alexeyinkin** [comment]
R: @pabloem

**Comment by alexeyinkin** [comment]
The Playground tests are broken after the MinimalWordCount update. I will fix it.

**Comment by pabloem** [comment]
lgtm

**Pull request merged**